stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -181,6 +181,7 @@ function _dump_function(@nospecialize(f), @nospecialize(t), native::Bool, wrappe
     # get the MethodInstance for the method match
     if !isa(f, Core.OpaqueClosure)
         world = Base.get_world_counter()
+        world == typemax(UInt) && error("code reflection cannot be used from generated functions")
```

Comment:
It might be okay to let it fall through to the error in the next function in some cases?
```
julia> Base._which(Base.signature_type(+, (Int, Int)); world=typemax(UInt))
ERROR: code reflection cannot be used from generated functions
```